geometry_msgs/Pose2D goalPose2D
---
std_msgs/Bool possible
